arch/arm/soc/st_stm32/stm32f1/Kconfig.defconfig.stm32f103x8

```diff
@@ -0,0 +1,18 @@
+# Kconfig - ST Microelectronics STM32F103RB MCU
```

Comment:
Please merge this file with other similar files to a single: 
arch/arm/soc/st_stm32/stm32f1/Kconfig.defconfig.stm32f103xx